### DISCUSSION TITLE
How to change page mtadata from a client component?

### DISCUSSION THREAD

**Discussion by 0xabdou** [vercel/next.js#61628] [App Router]
I need to be able to change the page title from a client component.
An example usecase is to show the number of unread notifications like some chat apps do (append something like `(1)` to the title).
I believe this was possible with `pages` by conditionally rendering a `Head` component, but not sure how to do it with `app`

**Comment by soxft** [comment]
```
export default function Page() {
  return (
    <>
      <title>Page title</title>

      <OtherComponent />
    </>
  )
}
```

https://stackoverflow.com/questions/76397721/how-to-set-title-of-a-client-component-in-next-13
